controllers/userController.js

```diff
@@ -36,9 +36,12 @@ router.get('/profile/:id', (req, res) => {
             id: req.params.id
         }
     }).then((data) => {
-        // console.log("this is data", data);
-        console.log("this is data", data.zipcode);
         res.render('profile', {
+            fullname: data.firstName + " " + data.lastName,
+            firstname: data.firstName,
+            lastname: data.lastName,
+            phone: data.mobile,
+            address: data.address,
```

Comment:
Passing all the profile data you have right now.  You'll likely need to add email, education and work experience to it (in the same table or in separate ones) if you want to put that data on the profile page.  Also, it looks great!

Reply:
Approved.